library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -128,6 +128,11 @@ class Group(val client: Client, private val libXMTPGroup: FfiGroup) {
         )
     }
 
+    fun isActive(): Boolean {
+        runBlocking { libXMTPGroup.sync() }
```

Comment:
I don't think there should be any reason you'd want a cached version of isActive. So syncing before hand.

Reply:
Depends on how people are using this. If you are running through a long list of conversations and calling is_active on all of them regularly, that can be pretty expensive.

Reply:
Interesting okay 🤔 Ya I could see that. So really only sync if they click on a group to make sure they can see it.